### DISCUSSION TITLE
使用了PAGSolidLayer给一个喇叭Icon进行了着色，但在开发完功能后，当出现不同音频之间切换播放时，会触发某种bug，导致原本的着色失效，变成未着色前的白色。技术也找不到原因

### DISCUSSION THREAD

**Discussion by bqduan** [Tencent/libpag#2499] [Q&A] [answered]
[speaker 任意色.zip](https://github.com/user-attachments/files/17143553/speaker.zip)

**Comment by kevingpqi123** [comment, marked as answer]
这里感觉还是使用层面的问题，pag 的相关接口很简单，就是设置颜色，然后 flush 或处于 play 状态下就可以生效。建议你们将相关颜色值打印出来进行分析
